src/LndClient/Data/Newtype.hs

```diff
@@ -251,3 +270,13 @@ defaultSyncGrpcTimeout = GrpcTimeoutSeconds 60
 
 defaultAsyncGrpcTimeout :: GrpcTimeoutSeconds
 defaultAsyncGrpcTimeout = GrpcTimeoutSeconds 3600
+
+toSat :: MSat -> Either LndError Sat
+toSat mSat = do
+  let mVal :: Word64 = coerce mSat
+  case divMod mVal 1000 of
+    (val, 0) -> Right $ Sat val
+    _ -> Left $ ToGrpcError "Cannot convert MSat to Sat"
```

Comment:
Let's have more info here
`"Cannot convert " <> show mSat <> " to Sat"`

Reply:
Okay

Reply:
done